main/src/ca/uwaterloo/flix/api/lsp/LspServer.scala

```diff
@@ -86,6 +86,13 @@ object LspServer {
     private val flixTextDocumentService = new FlixTextDocumentService(this, flixLanguageClient)
     private val flixWorkspaceService = new FlixWorkspaceService(this, flixLanguageClient)
 
+    /**
+      * The trigger characters for completion.
+      * By default, the client will only trigger completion requests on [a-zA-Z].
+      * These are the additional trigger characters.
+      */
+    private val triggerChars = List("#", ".", "/", "?")
```

Comment:
I would put these in `object LspServer `

Also constants should be uppercase: `TriggerChars`.

(Its possible you cannot keep it private, but thats OK)